pkg/service/service.go

```diff
@@ -43,13 +45,21 @@ func PreInitJiraAPI(s Service, params configuration.JiraAPIResourceParameters, h
 	return api, nil
 }
 
-func Execute(s Service, params configuration.JiraAPIResourceParameters) error {
+func Execute(s Service, params configuration.JiraAPIResourceParameters, lastStep bool) error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function Execute should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4860757)